core/trino-main/src/main/java/io/trino/dispatcher/DispatchManager.java

```diff
@@ -22,6 +22,7 @@
 import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Context;
 import io.trino.Session;
```

Comment:
Please extend commit message, e.g: what was the problem?

Reply:
It is described in the PR description, but I now copied it to the commit extended message.